Cargo.lock

```diff
@@ -11095,6 +11095,7 @@ dependencies = [
  "futures 0.3.28",
  "gpui",
  "indoc",
+ "itertools 0.11.0",
```

Comment:
I don't think you need this. It's fixed on `main`.

Reply:
Cargo really insists that I do need this, but okay, I've forcefully removed it for now